File: Java Programming/Fibonacci_series.java
class Fibonacci_series {
    
    public static void main(String [] args)
    {
        int a=0,b=1,c,r=10,i;
        System.out.println("The Range : "+r);
        System.out.print(""+a+" "+b);
        for(i=0;i<r;i++)
        {
            c=a+b;
            System.out.print(" "+c);
            a=b;
            b=c;
        }
    }
}

File: C Programming/Fibonacci_Series.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int a=0,b=1,c,r,i;
    printf("Enter the Range :");
    scanf("%d",&r);
    printf("%d %d",a,b);
    for(i=0;i<r;i++)
    {
        c=a+b;
        printf(" %d",c);
        a=b;
        b=c;
    }
}